transport/owr_session.c

```diff
@@ -148,6 +148,7 @@ static void owr_session_set_property(GObject *object, guint property_id, const G
             g_free(priv->dtls_certificate);
             priv->dtls_certificate = NULL;
         }
+        GST_DEBUG_OBJECT(OWR_SESSION(object), "cert in owr_session.c: %s\n", priv->dtls_certificate);
```

Comment:
As mentioned you don't need to mention the source file in the log message, it will be included.